content/post/057-ipfs-desktop-0-9.md

```diff
@@ -0,0 +1,65 @@
+---
+date: 2019-09-13
+url: 057-ipfs-desktop-0-9
+title: IPFS Desktop 0.9 released
+author: Henrique Dias
+---
+
+IPFS Desktop has come a long way to being what it is nowadays. We are here to cellebrate and announce the [release of 0.9.0](https://github.com/ipfs-shipyard/ipfs-desktop/releases) and, with that, tell a bit about the story of this application and, of course, the new features.
+
+## Once upon a time...
+
+It was back in May 2015 when Juan Benet pushed a first so-called "Initial Commit" to start a project at the time named "ipfs-electron". The purpose of the app at the time was quite simple and that hasn't changed much with time: providing a long running daemon, alongside with a GUI for managing it.
+
+> TODO: add timeline
+
+Regarding the application's name, it went from "ipfs-electron" to "IPFS Native Application", then to "IPFS Station", it being a much more fancier name. However, we changed it again to "IPFS Desktop" and that's where we are right now!
+
+> TODO: screenshots of older versions
+
+## IPFS Desktop 0.9.0
+
+The newest version of IPFS Desktop, 0.9.2 - yes, we already released two patch versions! - brings more stability and some new experimental features. Although it is not a completely revamped application, such as what happened with 0.7.0, I believe this version deserves an overview of the current (new) features and plans for the future.
+
+### Browse the wild...
+
+With the recent update to the Web UI, we are now capable of navigating throug the whole IPFS system. Previously, you could only access your 'Files' - Mutable File System for those who enjoy technical terms. Now, you can navigate not only to any `/ipfs/` or `/ipns/` path, but also to your pins!
+
+![Browing /ipns/ipns.io](/057-ipfs-desktop-0-9/browse-wild.png)
+
+### Use npm on ipfs
+
+In addition, we are introducing a new experiments section on the 'Settings' page which will allow you to enable new experimental features as they come in. To cellebrate this new section, we are adding [npm on ipfs](https://github.com/ipfs-shipyard/npm-on-ipfs) to it! This is a project that allows you to use `ipfs-npm` command to install your Node.js modules through the IPFS network!
+
+![Experiments section](/057-ipfs-desktop-0-9/npm-ipfs.png)
+
+**Please note** that this feature is intended for those who have Node.js installed on their system. Moreover, the activation will only work if your setup does not require additional permissions for executing `npm install -g`.
+
+### Add `ipfs` command line tools to your system
+
+Adding `ipfs` command line tools to your system is not a new feature and it was introduced in 0.8.0. Although, it's always worth noting that you are a click away of having the `ipfs` command available from your command line! Just go to Settings and enable "IPFS command line tools"!
```

Comment:
```suggestion
Adding `ipfs` command line tools to your system was actually introduced in 0.8.0, but it's always worth noting that you are just a click away of having the `ipfs` command available from your command line! Just go to Settings and enable "IPFS command line tools"!
```

Reply:
Another sentence or two about the importance of this (aka you don't have to worry about configuring your PATH variable or any confusing stuff to start using `ipfs` on the command line after installing desktop and enabling these tools!)